packages/cli/doc-plugin-preview/mdx-meta-loader.cjs

```diff
@@ -0,0 +1,61 @@
+const { join } = require('path');
+const { routeMeta } = require('./dist');
+
+module.exports = async function (source) {
+  const callback = this.async();
+  const { createProcessor } = await import('@mdx-js/mdx');
+  const { visit } = await import('unist-util-visit');
```

Comment:
Can we use `await import` in a .cjs file?

Reply:
yes